Documentation/contributor-guide/logging.md

```diff
@@ -1,29 +1,33 @@
 # Logging Conventions
 
-etcd uses the [capnslog][capnslog] library for logging application output categorized into *levels*. A log message's level is determined according to these conventions:
+etcd uses the [zap][zap] library for logging application output categorized into *levels*. A log message's level is determined according to these conventions:
 
-* Error: Data has been lost, a request has failed for a bad reason, or a required resource has been lost
+* Debug: Everything is still fine, but even common operations may be logged, and less helpful but more quantity of notices. Usually not used in production.
   * Examples:
-    * A failure to allocate disk space for WAL
+    * Send a normal message to a remote peer
+    * Write a log entry to disk
+
+* Info: Normal, working log information, everything is fine, but helpful notices for auditing or common operations.
```

Comment:
Consider: Should rather not be logged more frequently than once per a few seconds in normal server's operation.

Reply:
Thanks @ptabor Done.